contracts/staking/BaseShareField.sol

```diff
@@ -64,20 +64,17 @@ contract BaseShareField is DAOContract{
         }
         FundManager fm = FundManager(nameService.getAddress("FUND_MANAGER"));
         (uint rewardsPerBlock, uint blockStart, uint blockEnd, bool isBlackListed) = fm.getStakingReward(address(this));
-        if(block.number >= blockStart && blockEnd>= block.number && isBlackListed == false){
+        if(block.number >= blockStart && blockEnd>= block.number){
             uint256 multiplier = block.number.sub(lastRewardBlock);
```

Comment:
lastRewardBlock  must be  blockStart, so either initialize it to blockStart or make sure it is >=